CodeKata/NameToBanjoStrings.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeKata
+{
+    public class NameToBanjoStrings
+    {
+        public static string AreYouPlayingBanjo(string name)
+        {
+            if (name[0] == 'R' || name[0] == 'r')
```

Comment:
You can use ToLower method to make name[0] always lower case, so you wan't need to use alternative condition.